src/components/requests/drawing.vue

```diff
@@ -9,27 +15,38 @@
       channel.postRequest('DrawTools.StartDrawingRequest', data);"></code-component>
     </div>
     <div id="StopDrawingRequest">
-      <button class="btn btn-primary exampleready" @click="stopDrawing">DrawTools.StopDrawingRequest - cancel current drawing</button>
+      To stop current drawing progress send <code>StopDrawingRequest</code>.
+      <button class="btn btn-primary exampleready" @click="stopDrawing">DrawTools.StopDrawingRequest</button>
       <div>
         <a id="stopDrawing" href="" @click="getLink">To the documentation</a>
       </div>
       <code-component snippet="var data = ['my functionality id'];
       channel.postRequest('DrawTools.StopDrawingRequest', data);"></code-component>
     </div>
     <div id="StopDrawingRequestClear">
+      To clear drawing from the map send <code>StopDrawingRequest</code> with second member of data parameter given to StopDrawingRequest set as true.
       <button class="btn btn-primary exampleready" @click="stopDrawingClear">DrawTools.StopDrawingRequest - clear drawings</button>
       <code-component snippet="var data = ['my functionality id', true];
       channel.postRequest('DrawTools.StopDrawingRequest', data);"></code-component>
     </div>
+    <div id="DrawingEvent">
```

Comment:
Is id needed here? I've been removing them from all of the examples as they are not used but mostly complicate the code.

Reply:
Deleted ids and refs.